ui/selectors/selectors.js

```diff
@@ -490,6 +490,24 @@ export function getAllTokens(state) {
   return state.metamask.allTokens;
 }
 
+/**
+ * Get a flattened list of all ERC-20 tokens owned by the user.
+ * Includes all tokens from all chains and accounts.
+ *
+ * @returns {Object[]} All ERC-20 tokens owned by the user in a flat array.
```

Comment:
Whoops, my suggestion broke it. We used to prefer `Object` but our lint rules require it to be `object` now instead
```suggestion
 * @returns {object[]} All ERC-20 tokens owned by the user in a flat array.
```